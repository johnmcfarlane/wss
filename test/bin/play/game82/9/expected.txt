 39  1,3  - RuPEES
 36  1,3  - RuPEE
 36  1,3  - SuPER
 36  1,3  - SuPRA
 30  3,8  - IPS
 30  1,3  - PuREES
 30  1,3  - RuPA
 30  2,8  - SIP
 30  1,3  - SuPA
 30  1,3  - SuPE
 30  1,3  - SuPR
 28  2,8  | SOPE
 27  1,10 - APA
 27  1,10 - EPA
 27  1,3  - PuREE
 27  1,3  - PuRSE
 27  1,3  - SuP
 24 10,2  | APEEK
 24  1,8  - APIS
 24  7,3  | ARABS
 24  2,9  | OPA
 24  2,9  | OPE
 24 10,2  | SAPEK
 24  2,8  | SOP
 24 10,2  | SPARK
 24 10,2  | SPEAK
 23  7,4  - RAPES
 23  7,4  - REAPS
 23  7,4  - REPAS
 23  4,4  | REPOSE
 22  7,4  - RAPE
 22  7,4  - RAPS
 22  7,4  - RASP
 22  7,4  - REAP
 22  7,4  - REPS
 22  7,4  - SEEP
 21  7,5  | ABS
 21  3,7  - JOINS
 21  1,3  - PuRE
 21  1,3  - PuRS
 21  7,4  - RAP
 21  7,4  - REP
 21  7,4  - SAP
 21  7,4  - SEP
 20  2,9  | OP
 20  2,10 - PA
 20  7,4  - REESE
 20  7,4  - RESEE
 20 10,1  | RESEEK
 20  7,4  - SAREE
 20  5,9  | SSP
 19  8,8  | EPA
 19  4,6  | POSE
 19  7,4  - RASE
 19  7,4  - REES
 19  1,7  | REMAP
 19  5,6  | RIP
 19  7,4  - SARE
 19  7,4  - SEAR
 19  7,4  - SEER
 19  7,4  - SERA
 19  7,4  - SERE
 18 10,2  | AREEK
 18  1,3  - AuRES
 18  1,4  | PASE
 18  4,4  | PASO
 18  1,4  | PEER
 18  1,4  | PEES
 18  1,4  | PESA
 18  4,4  | PESO
 18  2,8  - PIR
 18  2,8  - PIS
 18  4,4  | PRAO
 18  1,3  - PuA
 18  1,3  - PuR
 18  1,3  - PuS
 18  7,4  - RAS
 18  7,4  - REA
 18 10,2  | REASK
 18  7,4  - REE
 18 10,2  | REESK
 18  7,4  - RES
 18  7,4  - SAE
 18  7,4  - SEA
 18  7,4  - SEE
 18  7,4  - SER
 17  4,4  | AEROSE
 17  1,9  | MAP
 17  1,4  | PAS
 17  1,4  | PEE
 17  1,6  | PERMS
 17  1,4  | PES
 17  1,6  | PRAMS
 17  4,5  | PROSE
 17  7,4  - RE
 17  1,4  | SAP
 17  1,4  | SEP
 16  8,4  - APER
 16  8,4  - APES
 16  8,4  - APSE
 16  8,4  - EPEE
 16  1,8  - EPI
 16  1,6  | PERM
 16  1,6  | PRAM
 16  1,6  | PREM
 16  5,6  | RIA
 16  5,6  | RIE
 16  4,3  | SARPO
 15  8,4  - APE
 15  1,5  | APE
 15  8,4  - APR
 15  1,5  | APR
 15  8,4  - ASP
 15  8,4  - EPA
 15  1,5  | EPA
 15  8,4  - EPE
 15  1,5  | EPE
 15  1,10 - ERA
 15  1,11 - PASE
 15  1,11 - PASS
 15  1,11 - PESA
 15  1,11 - PESS
 15  4,4  | REPO
 15  1,4  - RISP
 15  4,4  | SAPO
 15  6,3  | SAZEN
 15  1,4  | SEEP
 14  4,5  | AROSE
 14  8,7  - AX
 14  4,5  | EROSE
 14  8,5  | GOA
 14  1,11 - PAS
 14  1,11 - PES
 14  1,11 - PRS
 14  1,6  | SEAMER
 14  1,6  | SEEMER
 13  8,4  - ARES
 13  8,4  - ARSE
 13  8,4  - EARS
 13  8,4  - EASE
 13  8,4  - ERAS
 13  8,4  - ERSE
 13  1,4  | PA
 13  4,3  | PAREO
 13  4,3  | PASEO
 13  1,4  | PE
 13  2,2  - QERE
 13  1,7  | RAMPS
 13  4,6  | ROSE
 13  1,6  | SPAM
 12  8,4  - AER
 12  8,4  - AES
 12  1,8  | AMPS
 12  8,4  - ARE
 12  8,4  - ARS
 12  1,5  | ASP
 12  1,6  - ATAP
 12  8,4  - EAR
 12  8,4  - EER
 12  2,8  - EIR
 12  8,4  - ERA
 12  8,4  - ERE
 12  8,4  - ERS
 12  8,4  - ESE
 12  1,3  - EuRE
 12  3,8  - IRS
 12  8,2  | PARGO
 12  1,3  - RuES
 12  1,3  - RuSA
 12  1,3  - RuSE
 12  1,4  | SEER
 12  2,8  - SIR
 12  1,6  - STAP
 12  1,3  - SuER
 12  1,3  - SuRA
 12  1,3  - SuRE
 11  8,4  - AE
 11  1,4  | AES
 11  7,8  | APER
 11  7,8  | APES
 11  8,4  - AR
 11  6,11 - ARED
 11  8,4  - AS
 11  8,4  - EE
 11  7,8  | EPEE
 11  8,4  - ER
 11  8,4  - ES
 11  1,6  | PA
 11 10,3  | PARK
 11 10,3  | PASK
 11  1,6  | PE
 11 10,3  | PEAK
 11 10,3  | PEEK
 11 10,3  | PERK
 11  4,6  | PO
 11  7,8  - PRF
 11  1,4  | SAE
 11  1,4  | SEE
 11 10,3  | SPAK
 11  2,6  - TAP
 11  2,3  - uPAS
 10  7,8  | APE
 10  7,8  | EPE
 10  1,7  | PAMS
 10  7,9  | PEA
 10  7,9  | PEE
 10  7,9  | PER
 10  7,9  | PES
 10  2,8  - PI
 10  1,4  - PIERS
 10  2,8  | PO
 10  1,7  | RAMEE
 10  1,7  | RAMP
 10  1,4  - RIPES
 10  1,7  | SAMP
 10  1,6  | SEAM
 10  1,6  | SEEM
 10  1,7  | SEMEE
 10  1,4  - SIPER
  9  1,5  | AER
  9  4,4  | AERO
  9  1,5  | AES
  9  1,8  | AMP
  9  1,11 - APSE
  9  1,3  - AuS
  9  1,5  | EER
  9  1,8  | EMES
  9  1,8  | EMP
  9  1,5  | ESE
  9  2,4  - IPSE
  9  3,8  - IS
  9  4,7  | OSE
  9  8,9  | PA
  9  1,7  | PAM
  9  7,9  | PE
  9  1,4  - PIAS
  9  1,4  - PIER
  9  1,4  - PIES
  9  1,4  - PISA
  9  1,4  - PISE
  9  4,5  | PRO
  9  8,9  - Po
  9  1,11 - RASP
  9  1,4  - RIPE
  9  1,4  - RIPS
  9  1,7  | RPM
  9  1,3  - RuA
  9  1,3  - RuE
  9  1,3  - RuS
  9  8,2  | SARGO
  9  7,2  | SERAB
  9  4,4  | SERO
  9  1,4  - SIPE
  9  1,3  - SuE
  9  1,3  - SuR
  9  2,3  - uPS
  8  1,5  | AE
  8  6,9  - AE
  8  6,9  - AR
  8  7,3  | ARAB
  8  7,8  - ARF
  8  8,3  | ARGO
  8  1,5  | AS
  8  1,5  | EE
  8  6,9  - EE
  8  6,9  - ER
  8  7,8  - ERF
  8  8,3  | ERGO
  8  1,5  | ES
  8  2,4  - IPR
  8  2,4  - IPS
  8  1,9  | MAE
  8  1,9  | MAR
  8  1,9  | MAS
  8  1,9  | MEA
  8  1,9  | MER
  8  1,9  | MES
  8  1,9  | MRS
  8  7,8  | PEER
  8  7,8  | PEES
  8  7,8  | PREE
  8  7,8  | PRES
  8  7,11 - RAD
  8 10,3  | REAK
  8  7,11 - RED
  8 10,3  | REEK
  8  7,8  - REF
  8  8,3  | REGO
  8  7,11 - SAD
  8  8,3  | SAGO
  8 10,3  | SARK
  8 10,3  | SEAK
  8  7,11 - SED
  8 10,3  | SEEK
  8  7,8  | SEEP
  8  8,3  | SEGO
  8  2,3  - uP
  8  2,3  - uREASE
  7  1,4  | AE
  7  8,4  | AGO
  7 10,4  | ARK
  7  1,7  | ARMS
  7 10,4  | ASK
  7  8,8  - EF
  7  8,4  | EGO
  7 10,4  | ERK
  7  1,7  | ERMA
  7  1,7  | ERME
  7  9,8  - FEU
  7  7,8  | PEE
  7  7,8  | PRE
  7  1,7  | RAME
  7  1,7  | RAMS
  7  1,7  | REMS
  7  7,9  | REP
  7  8,9  | SAE
  7 10,4  | SAK
  7  1,7  | SAME
  7  1,7  | SEME
  7  7,9  | SEP
  7  1,4  - SIREE
  6  6,11 - AE
  6  1,4  - AIRE
  6  1,4  - AIRS
  6  6,11 - AR
  6  1,7  | ARM
  6  4,5  | ARO
  6  1,11 - ARSE
  6  6,11 - AS
  6  2,11 - ASP
  6  1,11 - ASSE
  6  1,7  | EAM
  6  1,11 - EASE
  6  6,11 - EE
  6  1,4  - EIRE
  6  1,8  | EME
  6  1,8  | EMS
  6  6,11 - ER
  6  1,11 - ERSE
  6  6,11 - ES
  6  1,11 - ESSE
  6  1,6  - ETAS
  6  2,4  - IARE
  6  2,4  - IRES
  6  1,4  - PIA
  6  1,4  - PIE
  6  1,4  - PIR
  6  1,4  - PIS
  6  1,6  - PTA
  6  6,9  | PTA
  6  6,9  | PTE
  6  6,9  | PTS
  6  7,4  | RAB
  6  1,7  | RAM
  6  1,11 - RASE
  6  1,7  | REM
  6  1,4  - RIAS
  6  1,4  - RIES
  6  1,4  - RIP
  6  1,4  - RISE
  6  1,8  | RMS
  6  7,4  | SAB
  6  1,7  | SAM
  6  4,5  | SAO
  6  1,7  | SEM
  6  1,4  - SIER
  6  1,4  - SIP
  6  1,4  - SIRE
  6  1,8  | SMA
  6  2,11 - SSP
  6  2,3  - uRASE
  6  2,3  - uREAS
  6  2,3  - uRSAE
  5  8,10 | AA
  5  8,11 - AD
  5  8,10 | AE
  5  1,11 - AES
  5  1,8  | AM
  5  7,8  | ARES
  5  1,11 - ARS
  5  1,11 - ASS
  5  8,11 - ED
  5  1,6  | EE
  5  1,8  | EM
  5  4,6  | EO
  5  1,6  | ER
  5  1,11 - ERS
  5  1,6  | ES
  5  1,11 - ESS
  5  2,4  - IRA
  5  2,4  - IRE
  5  2,4  - IRS
  5  2,4  - ISE
  5  1,9  | MA
  5  1,9  | ME
  5  1,4  - PI
  5  1,11 - RAS
  5  7,8  | REES
  5  1,11 - RES
  5  7,8  | SEER
  5  4,6  | SO
  5  2,6  - TAE
  5  2,6  - TAS
  5  2,3  - uREA
  5  2,3  - uRSA
  5  2,3  - uSAR
  5  2,3  - uSEE
  5  2,3  - uSER
  4  7,9  | AER
  4  7,9  | AES
  4  7,8  | ARE
  4  7,9  | EER
  4  7,8  | ERE
  4  7,8  | ESE
  4  7,9  | REA
  4  7,8  | REE
  4  7,9  | REE
  4  7,9  | RES
  4  7,8  | SAE
  4  7,9  | SEA
  4  7,8  | SEE
  4  7,9  | SEE
  4  7,9  | SER
  4  2,8  - SI
  4  2,8  | SO
  3  7,9  | AE
  3  1,4  - AIR
  3  1,4  - AIS
  3  1,5  - AL
  3  2,11 - ASS
  3  6,9  | ATE
  3  7,9  | EE
  3  1,4  - EIR
  3  1,5  - EL
  3  2,11 - ESE
  3  2,11 - ESS
  3  1,6  - ETA
  3  6,9  | ETA
  3  7,9  | RE
  3  1,4  - RIA
  3  1,4  - RIE
  3  6,9  | RTE
  3  1,4  - SIA
  3  1,4  - SIE
  3  1,4  - SIR
  3  1,6  - STA
  3  6,9  | STA
  3  6,9  | STR
  3  2,3  - uRA
  3  2,3  - uRS
  3  2,3  - uSE
  2  1,4  - AI
  2  2,11 - AS
  2  6,9  | AT
  2  7,10 | EE
  2  7,10 | ER
  2  7,10 | ES
  2  2,11 - ES
  2  6,9  | ET
  2  2,4  - IS
  2  5,7  | IS
  2  1,4  - SI
  2  6,10 | TA
  2  6,10 | TE
  2  2,3  - uS
