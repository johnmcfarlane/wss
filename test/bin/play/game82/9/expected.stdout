 39  1,3  - RuPEES
 36  1,3  - RuPEE
 36  1,3  - SuPER
 36  1,3  - SuPRA
 30  1,3  - PuREES
 30  2,8  - SIP
 30  1,3  - SuPE
 27  1,3  - PuREE
 27  1,3  - PuRSE
 27  1,3  - SuP
 24 10,2  | APEEK
 24  2,9  | OPE
 24  2,8  | SOP
 24 10,2  | SPARK
 24 10,2  | SPEAK
 23  4,4  | REPOSE
 22  7,4  - SEEP
 21  7,5  | ABS
 21  3,7  - JOINS
 21  1,3  - PuRE
 21  1,3  - PuRS
 21  7,4  - SAP
 20  2,9  | OP
 20  2,10 - PA
 20 10,1  | RESEEK
 20  7,4  - SAREE
 19  4,6  | POSE
 19  1,7  | REMAP
 19  5,6  | RIP
 19  7,4  - SEAR
 19  7,4  - SEER
 19  7,4  - SERA
 19  7,4  - SERE
 18  1,3  - AuRES
 18  1,4  | PEER
 18  1,4  | PEES
 18  4,4  | PESO
 18  2,8  - PIS
 18  1,3  - PuR
 18  1,3  - PuS
 18  7,4  - SAE
 18  7,4  - SEA
 18  7,4  - SEE
 18  7,4  - SER
 17  1,9  | MAP
 17  1,4  | PEE
 17  4,5  | PROSE
 16  8,4  - APER
 16  8,4  - APES
 16  8,4  - APSE
 16  8,4  - EPEE
 16  5,6  | RIA
 15  8,4  - APE
 15  8,4  - ASP
 15  1,10 - ERA
 15  1,11 - PASE
 15  1,11 - PASS
 15  4,4  | REPO
 15  1,4  | SEEP
 14  7,9  - APo
 14  4,5  | AROSE
 14  8,7  - AX
 14  4,5  | EROSE
 14  8,5  | GOA
 14  1,11 - PAS
 14  1,11 - PES
 13  8,4  - ARES
 13  8,4  - ARSE
 13  8,4  - EARS
 13  8,4  - EASE
 13  8,4  - ERAS
 13  1,4  | PA
 13  4,3  | PAREO
 13  4,3  | PASEO
 13  1,7  | RAMPS
 13  4,6  | ROSE
 12  1,8  | AMPS
 12  4,5  | APO
 12  8,4  - ARE
 12  8,4  - ARS
 12  1,6  - ATAP
 12  8,4  - EAR
 12  8,4  - ERA
 12  8,4  - ERE
 12  8,4  - ERS
 12  8,2  | PARGO
 12  1,3  - RuES
 12  1,3  - RuSE
 12  1,4  | SEER
 12  2,8  - SIR
 12  1,3  - SuER
 12  1,3  - SuRA
 12  1,3  - SuRE
 11  8,4  - AE
 11  7,8  | APER
 11  7,8  | APES
 11  8,4  - AR
 11  8,4  - AS
 11  7,8  | EPEE
 11  8,4  - ER
 11  8,4  - ES
 11 10,3  | PARK
 11 10,3  | PEAK
 11 10,3  | PEEK
 11 10,3  | PERK
 11  4,6  | PO
 11  1,4  | SAE
 11  1,4  | SEE
 11  2,6  - TAP
 11  2,3  - uPAS
 10  7,8  | APE
 10  1,7  | PAMS
 10  7,9  | PEA
 10  7,9  | PEE
 10  7,9  | PER
 10  7,9  | PES
 10  2,8  - PI
 10  1,4  - PIERS
 10  2,8  | PO
 10  1,7  | RAMEE
 10  1,7  | RAMP
 10  1,4  - RIPES
 10  1,7  | SAMP
  9  4,4  | AERO
  9  1,8  | AMP
  9  1,11 - APSE
  9  1,8  | EMES
  9  3,8  - IS
  9  4,7  | OSE
  9  8,9  | PA
  9  1,7  | PAM
  9  1,4  - PIAS
  9  1,4  - PIER
  9  1,4  - PIES
  9  4,5  | PRO
  9  8,9  - Po
  9  1,11 - RASP
  9  1,4  - RIPE
  9  1,4  - RIPS
  9  1,3  - RuE
  9  8,2  | SARGO
  9  1,4  - SIPE
  9  1,3  - SuE
  9  2,3  - uPS
  8  1,5  | AE
  8  6,9  - AR
  8  7,8  - ARF
  8  6,9  - ER
  8  8,3  | ERGO
  8  1,9  | MAE
  8  1,9  | MAR
  8  1,9  | MAS
  8  7,8  | PEER
  8  7,8  | PEES
  8  7,8  | PREE
  8  7,11 - RAD
  8  7,11 - RED
  8 10,3  | REEK
  8  7,8  - REF
  8  7,11 - SAD
  8  8,3  | SAGO
  8 10,3  | SARK
  8 10,3  | SEEK
  8  7,8  | SEEP
  8  8,3  | SEGO
  8  2,3  - uP
  8  2,3  - uREASE
  8  2,3  - uRPS
  7  1,4  | AE
  7  8,4  | AGO
  7 10,4  | ARK
  7  1,7  | ARMS
  7 10,4  | ASK
  7 10,4  | EEK
  7  8,8  - EF
  7  8,4  | EGO
  7  9,8  - FEU
  7  7,8  | PEE
  7  1,7  | RAMS
  7  1,7  | REMS
  7  7,9  | REP
  7  8,9  | SAE
  7  1,7  | SAME
  7  1,7  | SEME
  7  1,4  - SIREE
  6  1,4  - AIRS
  6  6,11 - AR
  6  1,7  | ARM
  6  1,11 - ARSE
  6  6,11 - AS
  6  2,11 - ASP
  6  1,11 - EASE
  6  1,8  | EME
  6  1,8  | EMS
  6  6,11 - ER
  6  6,11 - ES
  6  1,6  - ETAS
  6  2,4  - IRES
  6  1,4  - PIA
  6  1,4  - PIE
  6  1,4  - PIS
  6  1,7  | RAM
  6  1,11 - RASE
  6  1,7  | REM
  6  1,4  - RIAS
  6  1,4  - RIP
  6  1,4  - RISE
  6  7,4  | SAB
  6  1,4  - SIP
  6  1,4  - SIRE
  6  2,3  - uRASE
  6  2,3  - uREAS
  6  2,3  - uRP
  6  2,3  - uRSAE
  5  8,10 | AA
  5  8,11 - AD
  5  8,10 | AE
  5  1,8  | AM
  5  7,8  | ARES
  5  1,11 - ARS
  5  1,11 - ASS
  5  8,11 - ED
  5  1,8  | EM
  5  1,6  | ER
  5  1,11 - ERS
  5  1,6  | ES
  5  1,11 - ESS
  5  2,4  - IRE
  5  1,9  | MA
  5  1,9  | ME
  5  1,4  - PI
  5  1,11 - RAS
  5  7,8  | REES
  5  1,11 - RES
  5  7,8  | SEER
  5  4,6  | SO
  5  2,6  - TAS
  5  2,3  - uREA
  5  2,3  - uRSA
  5  2,3  - uSER
  4  7,8  | ARE
  4  7,8  | ERE
  4  7,8  | REE
  4  7,9  | REE
  4  7,9  | RES
  4  7,8  | SAE
  4  7,9  | SEA
  4  7,8  | SEE
  4  7,9  | SEE
  4  7,9  | SER
  4  2,8  - SI
  4  2,8  | SO
  3  7,9  | AE
  3  1,4  - AIR
  3  1,4  - AIS
  3  1,5  - AL
  3  2,11 - ASS
  3  6,9  | ATE
  3  1,5  - EL
  3  2,11 - ESS
  3  1,6  - ETA
  3  6,9  | ETA
  3  7,9  | RE
  3  1,4  - RIA
  3  1,4  - SIR
  3  2,3  - uSE
  2  1,4  - AI
  2  2,11 - AS
  2  6,9  | AT
  2  7,10 | ER
  2  7,10 | ES
  2  2,11 - ES
  2  6,9  | ET
  2  2,4  - IS
  2  5,7  | IS
  2  1,4  - SI
  2  6,10 | TA
  2  6,10 | TE
  2  2,3  - uS
